library/src/main/java/com/rtchagas/pingplacepicker/ui/PlaceConfirmDialogFragment.kt

```diff
@@ -100,7 +104,15 @@ class PlaceConfirmDialogFragment : AppCompatDialogFragment(), PingKoinComponent
 
         if (resources.getBoolean(R.bool.show_confirmation_map)) {
             val staticMapUrl = getFinalMapUrl()
-            Picasso.get().load(staticMapUrl).into(contentView.ivPlaceMap)
```

Comment:
I also slightly polished code for retrieving place image and map. Now it should work better if there is no map/image.